src/axom/klee/tests/klee_geometry_operators.cpp

```diff
@@ -3,20 +3,21 @@
 //
 // SPDX-License-Identifier: (BSD-3-Clause)
 
+#include "axom/primal/geometry/Point.hpp"
+#include "axom/primal/geometry/Vector.hpp"
+
 #include "axom/klee/GeometryOperators.hpp"
 
-#include "axom/primal/geometry/Vector.hpp"
+#include "axom/klee/tests/KleeMatchers.hpp"
+#include "axom/klee/tests/KleeTestUtils.hpp"
 
-#include <cmath>
```

Comment:
Can you add a note why we are using the .h header since that is not a normal thing to do in our code?

Reply:
cmath should include math.h and be the C++ standard way.

Reply:
> cmath should include math.h and be the C++ standard way.

I couldn't agree more, but the issue appears to be out of our control, and keeps cropping up.
(See: https://github.com/search?q=repo%3ALLNL%2Faxom+cmath&type=commits)

The problem appears to be an ordering dependency:
  (a) `msvc` does not include defines from `<math.h>` like `M_PI` unless `_USE_MATH_DEFINES` is defined, and 
  (b) there is a guard in `<cmath>` that prevents it from loading `<math.h>` more than once.
  (c) we define `_USE_MATH_DEFINES` in `axom/config.h`, but run into trouble when something brings in `<cmath>` before we define `_USE_MATH_DEFINES`. Specifically here `gtest` was included before `axom/config.hpp` and it pulls in `<cmath>` internally in some of its files. 

So, it looks like we (and our users) can either require `axom/config.hpp` to be included before anything else (or at least to define `_USE_MATH_DEFINES`) and be diligent about this (and hope for the best), or we can include `<math.h>` (insted of (`<cmath>`) after including `axom/config.hpp`.
I don't think the former is a realistic/reasonable thing to expect of users, so I opted for the latter.

Reply:
> Can you add a note why we are using the .h header since that is not a normal thing to do in our code?

@white238 -- would you like me to add a comment to each `#include <math.h>` or do you think one would do?
Looks like we have about 20 of them -- https://github.com/search?q=repo%3ALLNL%2Faxom%20math.h&type=code

Reply:
That seems like a lot. I am not sure I care that much.

Could this be solved with a clang-query check to enforce we always include our config first?

Reply:
... and about 60 `#include <cmath>` -- https://github.com/search?q=repo%3ALLNL%2Faxom+cmath&type=code
(minus a few from this PR)!

(I only changed the ones that MSVC complained about.)

Reply:
> Could this be solved with a clang-query check to enforce we always include our config first?

Not sure. I suspect that would be more challenging than a few manual changes since `msvc` lets us know when `M_PI` is not defined.

Reply:
Thanks everyone. Since there's not yet a clear resolution and the code's already tested on all platforms including EAS3, I'm going to merge this as is and we can update the `cmath` vs. `math.h` includes in a follow-up PR, as necessary/desired.